package.json

```diff
@@ -18,6 +18,7 @@
     "format-check": "prettier '{src,e2e}/**/*(*.js|*.jsx|*.ts|*.tsx|*.graphql)' --check",
     "start": "NODE_OPTIONS='--preserve-symlinks' concurrently 'yarn start:tsc' 'tsx src/index.ts'",
     "start:tsc": "tsc -b -w --preserveWatchOutput",
+    "start-minimal": "NODE_OPTIONS='--preserve-symlinks' tsx src/index.ts",
```

Comment:
Kva er denne til for?

Reply:
Jeg syns den `tsc` kommandoen vi alltid kjører ved siden av er veldig støyete når jeg koder så denne er bare for å kunne få litt cleanere output.

Så kjører jeg heller tsc ved siden av selv.